Solutions/Auth0/Data Connectors/Auth0Connector/main.py

```diff
@@ -76,45 +68,98 @@ def __init__(self, domain, api_path, client_id, client_secret, audience):
         self.uri = self.domain + self.api_path
         self.token = None
         self.header = None
-
-    def get_log_events(self, config: dict) -> Tuple[str, List]:
+        self.retry = retry  
+
+    """This method is used to process and post the results to Log Analytics Workspace
+        Returns:
+            last_log_id : last processed eventId
+            events: last processed Events
+    """
+    def get_log_events(self, script_start_time, config: dict) -> Tuple[str, List]:
         self.token = self._get_token()
         logging.info(f'Token provided.')
         self.header = self._get_header()
         last_log_id = self._get_last_log_id(config)
         #last_log_id = "90020230126121002244690048186607762971591195832157732866"
         logging.info(f'\tLast log id extracted: {last_log_id}.')
         if last_log_id is None:
-            return '', []
+            #return '', []
+            self.update_statemarker_file(config, '', [])
+            return
         # first request
         params = {'from': last_log_id, 'take': '100'}
-        resp = requests.get(self.uri, headers=self.header, params=params)
```

Comment:
Have we tested this code for all the scenarios

Reply:
Yes, we get it verified by customer and attached screenshot from customer environment as we don't have test account.